opencensus/metrics/export/metric.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.metrics.export.metric_descriptor import MetricDescriptorType
+
+from opencensus.metrics.export.value import ValueLong
+from opencensus.metrics.export.value import ValueDouble
+from opencensus.metrics.export.value import ValueSummary
+from opencensus.metrics.export.value import ValueDistribution
+
+
+class Metric(object):
+    """A collection of time series data and label metadata.
+
+    This class implements the spec for v1 Metrics as of opencensus-proto
+    release v0.0.2. See opencensus-proto for details:
+
+        https://github.com/census-instrumentation/opencensus-proto/blob/24333298e36590ea0716598caacc8959fc393c48/src/opencensus/proto/metrics/v1/metrics.proto#33  # noqa
+
+    Defines a Metric which has one or more timeseries.
+
+    :type descriptor: class: '~opencensus.metrics.export.metric_descriptor.MetricDescriptor'  # noqa
+    :param descriptor: The metric's descriptor.
+
+    :type timeseries: list(:class: '~opencensus.metrics.export.time_series.TimeSeries')
+    :param timeseries: One or more timeseries for a single metric, where each
+    timeseries has one or more points.
+    """
+
+    def __init__(self, time_series, descriptor):
```

Comment:
nit: s/descriptor/metric_descriptor, also first param should be ```descriptor```.

Reply:
It's called `descriptor` in the proto def: https://github.com/census-instrumentation/opencensus-proto/blob/24333298e36590ea0716598caacc8959fc393c48/src/opencensus/proto/metrics/v1/metrics.proto#L37.